src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -404,6 +405,19 @@ export default class CanvasController {
     }
 
     /**
+     * Fill entire scope with the background color specified in the current theme
+     *
+     * @for CanvasController
+     * @method canvas_fill_background
+     * @param cc {Object} HTML5 canvas context
+     */
+    canvas_fill_background(cc) {
+        cc.fillStyle = this.theme.SCOPE.BACKGROUND;
+
+        cc.fillRect(0, 0, this.canvas.size.width, this.canvas.size.height);
```

Comment:
is it possible to move `this.canvas.size.width` to something like `this.size.width` or maybe even `this.width`?  I worry that these values are intertwined with other classes elsewhere in the project, though, so this may be a big ask.